### DISCUSSION TITLE
How to find and use zstd library?

### DISCUSSION THREAD

**Discussion by ghost** [JulNadeauCA/BSDBuild#11] [General]
With CMake, it's very simple: `find_package(zstd REQUIRED)`

**Comment by JulNadeauCA** [comment]
A new test module `zstd` was committed in https://github.com/JulNadeauCA/BSDBuild/commit/0ee0d05f2d21a6bc47449a1778356550b7a8bed3 and will be included in the upcoming 3.2 release.
Thank you for the suggestion! I'll be adding modules for your other suggested libraries shortly.
